react-components/src/architecture/concrete/pointsOfInterest/CreatePointsOfInterestWithDescriptionCommand.tsx

```diff
@@ -110,3 +113,18 @@ export class CreatePointsOfInterestWithDescriptionCommand extends CustomBaseInpu
     return true;
   }
 }
+
+function filterContentsAsString(contents: FieldContent[]): string[] {
+  return contents
+    .map((content) => {
+      if (
+        content.type === 'text' ||
+        content.type === 'comment' ||
+        content.type === 'commentWithButtons'
+      ) {
+        return content.content;
+      }
+      return undefined;
+    })
+    .filter((content) => !isUndefined(content));
```

Comment:
I think we have a `isDefined` in react-components - so you could write `.filter(isDefined)`

Reply:
ohh cool
done
https://github.com/cognitedata/reveal/pull/4929/commits/c0bebe9a063de9f6d742f5648af6bf4636363e34